src/components/Breeds.js

```diff
@@ -1,27 +1,62 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import "./Breeds.css";
 
-class Breeds extends React.Component {
-  render() {
-    return (
-      <div className="Breeds">
-        <h2 className="Breeds-title">Select a Breed</h2>
-        <p>
-          <select className="Breeds-select">
-            <option>beagle</option>
-            <option>bluetick</option>
-            <option>bullterrier-staffordshire</option>
-            <option>malinois</option>
-            <option>wolfhound-irish</option>
-          </select>
-        </p>
-        <img className="Breeds-image" src="http://via.placeholder.com/300x300" />
-        <p>
-          <button className="Breeds-button">Show me more!</button>
-        </p>
-      </div>
-    );
+const Breeds = () => {
+  const options = [
+    "beagle",
+    "bluetick",
+    "bullterrier/staffordshire",
+    "malinois",
+    "wolfhound/irish"
+  ];
+
+  const [selectedPhoto, setSelectedPhoto] = useState(null);
+  const [selectState, setSelectState] = useState(options[0]);
+
+  useEffect(() => {
+    fetchRandomPhoto(selectState);
+  }, [selectState]);
+
+  function fetchRandomPhoto(breed) {
+    fetch(`https://dog.ceo/api/breed/${breed}/images/random`)
+      .then((res) => res.json())
+      .then((data) => setSelectedPhoto(data.message))
+      .catch((err) => alert(err));
+  }
+
+  function handleSelect(e) {
+    setSelectState(e.target.value);
+  }
+
+  function handleClick() {
+    const randomBreed = options[Math.floor(Math.random() * options.length)];
+    setSelectState(randomBreed);
   }
-}
+
+  return (
+    <div className="Breeds">
+      <h2 className="Breeds-title">Select a Breed</h2>
+      <p>
+        <select className="Breeds-select" value={selectState} onChange={handleSelect}>
+          {options.map((breedName, index) => (
+            <option key={index} value={breedName}>
+              {breedName}
+            </option>
+          ))}
+        </select>
+      </p>
+      <img
+        className="Breeds-image"
+        src={selectedPhoto ? selectedPhoto : "http://via.placeholder.com/300x300"}
+        alt={selectedPhoto ? selectedPhoto.slice(30, 6) : "empty"}
```

Comment:
I'm not sure if `selectedPhoto.slice(30, 6)` will do what we expect here, I think this will always return an empty string? Is the intention to get the breed and image name? In which case `selectedPhoto.slice(30)` will work. If this is not the intention, maybe add a comment to say what is expected here from the slice